crates/ruff_linter/src/rules/ruff/rules/sequence_sorting.rs

```diff
@@ -434,14 +434,18 @@ impl MultilineStringSequenceValue {
             locator,
         );
 
+        let needs_trailing_comma = self.ends_with_trailing_comma
+            && first_non_trivia_token(TextSize::new(0), &postlude)
+                .map_or(true, |tok| tok.kind() != SimpleTokenKind::Comma);
```

Comment:
Do you mind adding a comment here to explain the logic? I think I understand now: we need the formatted value to end in a trailing comma, but if it's part of the postlude, it'll be inserted verbatim anyway.

Reply:
yeah, exactly. I'll add a comment!